src/WebJobs.Script/FileTraceWriter.cs

```diff
@@ -38,9 +40,17 @@ public FileTraceWriter(string logFilePath, TraceLevel level) : base(level)
             }
             else
             {
```

Comment:
What's the lifetime of `FileTraceWriter`? If it's created frequently, any chance of problems if the cleanup logic ends up executing many times at once (possibly across instances)?